src/app.py

```diff
@@ -54,9 +55,12 @@ def main():
     dev_file_builder_args = configurations.get_dev_file_builder_args()
     dev_factory_args = configurations.get_dev_factory_args()
 
+    current_time_string = datetime.now(timezone.utc).strftime("%H:%M:%S")
+
     for factory_definition in factory_definitions:
```

Comment:
I wonder if it makes more sense to instantiate a `google_drive_connector` at the start of each iteration of this loop using the `current_time_string`, then passing that connector object to the `instantiate_file_builder` method - would remove the responsibility of creating the connector from the `instantiate_file_builder` method and the function arguments would be a bit more readable at a glance.

More of a nitpick than a problem so maybe lets discuss when you're back